### ISSUE TITLE
Custom field mapping to Microsoft.VSTS.TCM.ReproSteps is ignored

### ISSUE THREAD

**Issue by contactsimonwilson** [solidify/jira-azuredevops-migrator#56]
Mapping any field other than description to Microsoft.VSTS.TCM.ReproSteps does not work as expected. Instead of copying the field details from the specified field (eg a custom repro steps field in Jira), the contents from the description field are used. 

This occurs because the description field is always mapped to the repro steps field for bugs by the first line in the MapLastDescription method inside the JiraMapper class.

```
  private void MapLastDescription(List<WiRevision> revisions, JiraItem issue)
        {
            var descFieldName = issue.Type == "Bug" ? "Microsoft.VSTS.TCM.ReproSteps" : "System.Description";
```

Sample config:
```
      {
        "source": "Steps to Reproduce",
        "source-type": "name",
        "target": "Microsoft.VSTS.TCM.ReproSteps",
        "for": "Bug"
      },
```

**Comment by MOlausson** [comment]
@contactsimonwilson appoligies for the delayed response here. What type of field are you using instead of repro step for your mapping? Agree it should not be hard-coded as it is and we'll adress it.

**Comment by beverts312** [comment]
This same code also causes custom field mapping description mappings to get overridden on non bug types.

**Comment by beverts312** [comment]
@MOlausson is the intention behind `MapLastDescription` just to try and make it so you dont have to define the Description/Steps to Reproduce mappings?

It seems like the easiest fix would be to just remove this method, I am running migrations with https://github.com/solidify/jira-azuredevops-migrator/blob/master/src/WorkItemMigrator/JiraExport/JiraMapper.cs#L135 commented out and everything works as expected.

**Comment by MOlausson** [comment]
@madkoo can you check this one in retrospect of the improvements you've made to field handling?

**Comment by MOlausson** [comment]
Fixed from v2.2.88.
